src/Network/Receive/ServerType0.pm

```diff
@@ -570,6 +570,9 @@ sub new {
 		'099D' => ['received_characters', 'v a*', [qw(len charInfo)]],
 		'099F' => ['area_spell_multiple2', 'v a*', [qw(len spellInfo)]], # -1
 		'09A0' => ['sync_received_characters', 'V', [qw(sync_Count)]],
+		'09A6' => ['banking_status', 'V2 x2',[qw(inbank reason)]], #rA : reason = 0
+		'09A8' => ['banking_status_deposit', 'v V3',[qw(reason inbank onhand)]],
```

Comment:
have more unpack string than vars to receive

Reply:
09A8 <Reason>W <Money>Q <balance>L (PACKET_ZC_ACK_BANKING_DEPOSIT)

WORD, QWORD, LONG

Reply:
FIXED :  c85ed1e